.travis.yml

```diff
@@ -36,8 +36,9 @@ notifications:
   email: false
 
 script:
-  - mvn -q test -Pspark-1.6
   - mvn -q test -Pspark-2.0
+  # test the spark-1.6 module only in this second run
+  - mvn -q test -Pspark-1.6 -Dtest=org.apache.spark.*
```

Comment:
yea, I think so.
IIUC the first tests the `Hivemall` core stuffs (e.g., core, nlp, and mixserv) and the spark-2.0 module.
The other tests the spark-1.6 module only because the `Hivemall` core stuff has already been tested in the first test.